manifests/stub/validate_addr.pp

```diff
@@ -0,0 +1,25 @@
+# Class: unbound::stub::validate_addr
+#
+# Validate address arg of unbound::stub
+#
+define unbound::stub::validate_addr {
+  $addr_arr = split($name, '@')
+
+  # check if the address arg is a valid ip address
+  if is_ip_address($addr_arr[0]) {
+
+    # multiple '@' signs invalid
+    if size($addr_arr) > 2 {
+      fail('unbound::stub: invalid address. too many at signs in it.')
```

Comment:
Thank you. I added it so please see https://github.com/K-DEN/puppet-unbound/commit/b1488bf039bbb0149c0e0773723132eda66bac50 .